src/clojure_tictactoe/game/game_rules.clj

```diff
@@ -0,0 +1,21 @@
+(ns clojure-tictactoe.game.game-rules
+  (:require [clojure-tictactoe.game.board :as board]))
+
+
+(defn assess-win 
+  [board]
+  (cond 
+    (= (board 0) (board 1) (board 2)) (board 0)
+    (= (board 3) (board 4) (board 5)) (board 3)
+    (= (board 6) (board 7) (board 8)) (board 6)
+    (= (board 0) (board 3) (board 6)) (board 0)
+    (= (board 1) (board 4) (board 7)) (board 1)
+    (= (board 2) (board 5) (board 8)) (board 2)
+    (= (board 0) (board 4) (board 8)) (board 0)
+    (= (board 2) (board 4) (board 6)) (board 2)))
```

Comment:
This works for now, but I'd like to see you refactor this later. It's pretty verbose and I bet you can get it to be simpler and clearer.

Reply:
I agree.

Reply:
+1. What's the plan for implementing the refactor for this?